scripts/build/termux_create_pacman_subpackages.sh

```diff
@@ -99,7 +99,7 @@ termux_create_pacman_subpackages() {
 			fi
 
 			if [ -n "$TERMUX_SUBPKG_CONFFILES" ]; then
-				tr ',' '\n' <<< "$TERMUX_SUBPKG_CONFFILES" | awk '{ printf "backup = ${TERMUX_PREFIX}/%s\n", $1 }'
+				tr ',' '\n' <<< "$TERMUX_SUBPKG_CONFFILES" | awk '{ printf "backup = data/data/com.termux/files/usr/%s\n", $1 }'
```

Comment:
Hardcoding the prefix is a bad idea. Fix so that TERMUX_PREFIX is expanded instead, with for example

```
tr ',' '\n' <<< "$TERMUX_SUBPKG_CONFFILES" | awk '{ printf "backup = '"$TERMUX_PREFIX"'/%s\n", $1 }"
```

Reply:
If you need to strip the first / you can use `${TERMUX_PREFIX:1}`

Reply:
Your way is not working.  I tried to fix it but nothing worked.

Reply:
Last character in my line should be `'` instead of `"`

Reply:
Even after this fix, there are still bugs.

Reply:
It should be TERMUX_PKG_CONFFILES on that line, [see the line before](https://github.com/termux/termux-packages/pull/7455/files#diff-c1aa2b5f55742357b6e1331f940431254a124e3b90a58a557d99f81972226d04R95)

Reply:
Thank